doc/command_ref.rst

```diff
@@ -398,7 +398,7 @@ Install Command
 ---------------
 
 ``dnf [options] install <spec>...``
-    Make sure that the given packages and their dependencies are installed on the system. Each ``<spec>`` can be
+    Dnf makes sure that the given packages and their dependencies are installed on the system. Each ``<spec>`` can be
```

Comment:
Now I get why you want the change. Use capital letters when you reference DNF and make the commit message more specific, please.